### PR TITLE
feat: shortcut to add and subtract five mins and set directly to number with number keys

### PR DESCRIPTION
**Pull request by niajane**
Not sure about the choice of keys (at first I thought h and l but h was taken)

### CODE REVIEW

**Comment by niajane** [comment]
just realised L is taken as well D: Maybe U and N but maybe this is getting too unintuitive

**Comment by ddorn** [comment]
Nice! 🥰
I think shift+j and shift+k would be the best. I also had the plan of using all the digits to reset it directly to this duration (and with shift, to 10 times the duration)

**Comment by niajane** [comment]
- remove j and u shortcuts
- shift+j/k for sub/add five mins
- number keys to set timer to that many mins; shift+number for timer number*10 mins
- pull out round up to minute functionality into function "round_time_up_to_minute"
- new file "event_key_utils" - let me know if you would prefer this in the same file

**Review by ddorn — COMMENTED** [review]
Amazing ! 🤩
I would prefer indeed to keep this in one file for now. Though I feel the need to split it slowly going up.

File: pucoti.py [hunk 1]
```diff
@@ -647,12 +654,19 @@ def save_last_purpose():
                     else:
                         scene = Scene.MAIN
                 elif event.key == pg.K_j:
-                    timer -= 60
+                    timer -= 60 * 5 if shift_is_pressed() else 60
                 elif event.key == pg.K_k:
-                    timer += 60
+                    timer += 60 * 5 if shift_is_pressed() else 60
+                elif event.key in NUMBER_KEYS:
```
Comment (ddorn) [inline pucoti.py:660]: I would also set `initial_duration` to the new duration here, so that reset uses the new duration. It seems a bit better

**Pull request merged**